diffblue.yml

```diff
@@ -1,5 +1,4 @@
 buildCmd: mvn compile
-testCmd: mvn test
```

Comment:
How does the product behave if the testCmd is not specified?

Reply:
When I tested it, it worked correctly in the sense that it just skipped over running tests and carried on with no coverage.

But having just retried at http://35.187.71.44/results/java-demo-no-existing-tests/feature~2Fremove-existing-tests, there is a problem with coverage stats, but I suspect it may be an unrelated issue.

Reply:
Thanks for checking.